grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
This particular test tests the fix this PR is about. It fails without the correction.